integration-tests/common.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```

Comment:
I'd like to have this doc (well it's almost empty ATM) displayed anywhere (and be able to link to it). I'm not sure whether it is possible to push generated doc directly into GitHub's wiki (it might be the solution).